andela/theoden-cfh
# feat(localization): Cards should be categorised by region

#### What does this PR do?

A player who starts a game should have the option of selecting a region for the game. This would ensure the cards are relevant to that particular geographical region.

#### Description of Task to be completed?

GIVEN I'm on the home page
WHEN I start a new game
THEN I should be given the option to choose which part of the world I am in
GIVEN I'm playing a game
WHEN Cards are drawn
THEN The questions and answers should be related to my region

How should this be manually tested?

After login into the application, the user sees a play with friends and play with strangers button. On clicking either button a modal pops up to inform the user that they are about to enter the gaming screen while also providing the option to select a region. Once the region is selected, the user clicks the start button and enters into the gaming screen. If the number of players are up to the required minimum when the game starts, the questions will be categorised for users from that region

Any background context you want to provide?

What are the relevant pivotal tracker stories?

Users should be able to create/start a new game
Cards should be categorised by region

Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/13855608/31441878-6dc3e57e-ae8c-11e7-8ff1-cea67227ab14.png)

<img width="1066" alt="screenshot 2017-10-11 11 36 55" src="https://user-images.githubusercontent.com/13855608/31436380-94049588-ae7a-11e7-8bd4-93aae9b7d8bd.png">
